pmmlserver/rockcraft.yaml

```diff
@@ -0,0 +1,61 @@
+# Based on https://github.com/kserve/kserve/blob/master/python/pmml.Dockerfile
+name: pmml-server
+summary: Pmml server for Kserve deployments
+description: "Kserve Pmml server"
+version: "v0.11.0_22.04_1"
+license: Apache-2.0
+base: ubuntu:22.04
+build-base: ubuntu:22.04
+platforms:
+    amd64:
+run-user: _daemon_
+services:
+  pmml-server:
+    override: replace
+    summary: "Pmml server service"
+    startup: enabled
+    command: python3.10 -m pmmlserver
+    environment:
+      PYTHONPATH: /pmmlserver:/kserve
+
+parts:
+  security-team-requirement:
+    plugin: nil
+    override-build: |
+      mkdir -p ${CRAFT_PART_INSTALL}/usr/share/rocks
+      (echo "# os-release" && cat /etc/os-release && echo "# dpkg-query" && \
+      dpkg-query -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W) \
```

Comment:
For 22.04 we can have a better way of collecting packages (from our best practices):
`dpkg-query --root=${CRAFT_PROJECT_DIR}/../bundles/ubuntu-22.04/rootfs/ -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W)`